pkg/modifycontroller/modify_volume.go

```diff
@@ -67,6 +67,16 @@ func (ctrl *modifyController) modify(pvc *v1.PersistentVolumeClaim, pv *v1.Persi
 		}
 	}
 
+	// Rollback infeasible errors for recovery
+	if pvc.Status.ModifyVolumeStatus != nil && pvc.Status.ModifyVolumeStatus.Status == v1.PersistentVolumeClaimModifyVolumeInfeasible {
```

Comment:
So I personally like large conditions like these to be behind functions which I can readily understand when reading this code again. I made a refactor of this - https://github.com/gnufied/external-resizer/commit/fa37fc16b2708edb0a4b604ecbbe71382a62f7c1 would you consider pulling this in?

Reply:
Yes! Let me update the PR and ping you again when tests pass etc.